packages/merkleize-metadata/src/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { type HexString } from "@polkadot-api/substrate-bindings"
+import { fromHex } from "@polkadot-api/utils"
+
+export const mergeUint8 = (inputs: Array<Uint8Array>): Uint8Array => {
+  const len = inputs.length
+  let totalLen = 0
+  for (let i = 0; i < len; i++) totalLen += inputs[i].byteLength
+  const result = new Uint8Array(totalLen)
+
+  for (let idx = 0, at = 0; idx < len; idx++) {
+    const current = inputs[idx]
+    result.set(current, at)
+    at += current.byteLength
+  }
+
+  return result
+}
```

Comment:
If we are already importing `fromHex`, we can import as well `mergeUint8`.

Reply:
unfortunately the `mergeUint8` that's currently on the `@polkadot-api/utils` can't be used for this... The reason being that the `inputs` come as a spread arg, and when there are too many items it crashes. It's in my TODO list to fix that. Whenever I get to it, then I will remove this one and use the fixed one.

Reply:
Sounds good!